packages/agent/src/test-managed-agent.ts

```diff
@@ -0,0 +1,243 @@
+
+import { Jose } from '@web5/crypto';
+import { Dwn } from '@tbd54566975/dwn-sdk-js';
+import { DidIonMethod, DidKeyMethod, DidResolver } from '@web5/dids';
+import { KeyValueStore, LevelStore, MemoryStore } from '@web5/common';
+import { MessageStoreLevel, DataStoreLevel, EventLogLevel } from '@tbd54566975/dwn-sdk-js/stores';
```

Comment:
React Native isn't able to import these, as they're named exports. Going to make a PR against the dwn repository to move these exports into the primary `dwn-sdk-js` package.

Reply:
PR here: https://github.com/TBD54566975/dwn-sdk-js/pull/477